app/javascript/react/screens/App/Overview/components/InfrastructureMappingsList/InfrastructureMappingsList.js

```diff
@@ -1,226 +1,551 @@
+                    );
+                    const targetClusterCount = this.destinationCount(
+                      mapping.transformation_mapping_items,
+                      'emscluster'
+                    );
+                    const sourceDatastoreCount = this.sourceCount(
+                      mapping.transformation_mapping_items,
+                      'storage'
+                    );
+                    const targetDatastoreCount = this.destinationCount(
+                      mapping.transformation_mapping_items,
+                      'storage'
+                    );
+                    const sourceLanCount = this.sourceCount(
+                      mapping.transformation_mapping_items,
+                      'lan'
+                    );
+                    const targetLanCount = this.destinationCount(
+                      mapping.transformation_mapping_items,
+                      'lan'
+                    );
+                    const associatedPlansCount =
+                      mapping.service_templates &&
+                      mapping.service_templates.length;
+
+                    const {
+                      targetClusters,
+                      targetDatastores,
+                      targetNetworks
+                    } = mapInfrastructureMappings(
+                      mapping.transformation_mapping_items,
+                      clusters,
+                      datastores,
+                      networks
+                    );
+
+                    return (
+                      <ListView.Item
+                        key={mapping.id}
+                        heading={mapping.name}
+                        description={
+                          <small>
+                            {__('Completed: ')}
+                            {moment(mapping.created_at).format(
+                              'MMMM Do YYYY, h:mm a'
+                            )}
+                          </small>
+                        }
+                        stacked
+                        compoundExpand
+                        compoundExpanded={mapping.expanded}
+                        onCloseCompoundExpand={() => this.closeExpand(mapping)}
+                        additionalInfo={[
+                          <ListView.InfoItem key={0}>
+                            <ListView.Expand
+                              expanded={
+                                mapping.expanded && mapping.expandType === 0
+                              }
+                              toggleExpanded={() => {
+                                this.toggleExpand(mapping, 0);
+                              }}
+                            >
+                              <Icon type="pf" name="network" />
+                              <div className="mappings-expand-label-group">
+                                <div className="mappings-expand-label">
+                                  <strong>{sourceLanCount}</strong>&nbsp;
```

Comment:
if we gotta have em, we gotta have em, but if there's a better way than sprinkling `&nbsp;` :-/ :trollface:

Reply:
i still think those are more explicit and we don't really have a standard on this... i.e. "how many spaces am I looking at..."

Reply:
what do you do elsewhere in MIQ?

Reply:
yeah it happens everywhere in miq... doesn't mean I like it 😏

Reply:
ok man - you win... my <3 of JSX supersedes my <3 of explicit spaces... 

JSX is really HTML6 ... a lotta folks just don't really admit it yet ;)

Reply:
❤️ ❤️ ❤️ ❤️ ❤️ ❤️